### PR TITLE
Practice-One - create Feature/init structure

### PR DESCRIPTION
**Pull request by Vinh2410**
Create init structure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+- Step 03: Move to folder which just cloned in your computer:
+
+```bash
+cd html-css-training
+```
+
+- Step 04: Next open folder practice-one
+
+```bash
+cd practice-one
```
Comment (dieule-fox) [inline README.md]: Merge step3 and step4 to only one step

**Comment by dieule-fox** [comment]
Add .editorconfig 
https://editorconfig.org/

**Comment by dieule-fox** [comment]
where is index.html file?

**Comment by dieule-fox** [comment]
Add .gitignore

**Comment by dieule-fox** [comment]
Move css to assets/ cause it also static assets

**Comment by dieule-fox** [comment]
Rename practice_one to practice-one

**Comment by dieule-fox** [comment]
Do we have pages/?

File: README.md (hunk 1 above)
Comment (Vinh2410) [reply README.md]: I fixed it

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
-# html_css_training
\ No newline at end of file
+## Target:
```
Comment (dieule-fox) [inline README.md]: Move README to under practice-one

**Pull request merged**